lib/flight_job/command.rb

```diff
@@ -203,8 +203,15 @@ def load_job(id)
     def assert_results_dir_exists(job)
       # NOTE: Jobs created with old versions of flight-job will not have a
       # results directory.
-      unless job.results_dir
-        raise MissingError, "The job did not report its results directory"
+      if !(job.results_dir && Dir.exists?(job.results_dir))
+        case job.state
+        when 'PENDING'
+          raise MissingError, 'Your job has not started, please try again latter...'
+        when *Job::RUNNING_STATES
+          raise MissingError, 'No job results found, please try again latter...'
+        else
+          raise MissingError, 'No job results found.'
+        end
```

Comment:
These error messages can also be printed directly by the `ListJobResultsCommand#run` method.  When printed by the `run` method they are coloured.  When raised here they aren't.  Can we get these coloured too please?

Reply:
I am not entirely sure why the colourisation isn't working. The top level error handler should be applying the default colours. This _may_ be an issue with your `TERM` setting as I can not reproduce it.

---


Regardless, I have collapsed the two sets of warnings/errors into a single method:
7f2ce35fef7732fdf6542cf7f9233dbfdf5113af